flang/lib/Lower/Bridge.cpp

```diff
@@ -1140,9 +1145,20 @@ class FirConverter : public Fortran::lower::AbstractConverter {
 
   void genFIR(const Fortran::parser::OpenMPConstruct &omp) {
     auto insertPt = builder->saveInsertionPoint();
+    localSymbols.pushScope();
     genOpenMPConstruct(*this, getEval(), omp);
-    for (auto &e : getEval().getNestedEvaluations())
+    // If loop is part of an OpenMP Construct then the OpenMP dialect
+    // workshare loop operation has already been created. Only the
+    // body needs to be created here and the do_loop can be skipped.
+    Fortran::lower::pft::Evaluation *curEval{nullptr};
```

Comment:
NIT: `curEval = nullptr` , Bridge code also try to follow MLIR coding style.

Reply:
And if you like, can drop the braces for `if-else`

Reply:
Can this all be reduced to 1 line with
```c++
Fortran::lower::pft::Evaluation *curEval = std::get_if<Fortran::parser::OpenMPLoopConstruct>(&omp.u) ? &getEval().getFirstNestedEvaluation() : &getEval();
```
?